frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -1214,6 +1219,12 @@ export const eventUsageLogic = kea<eventUsageLogicType>([
                 end_date: survey.end_date,
             })
         },
+        reportSurveyLaunched: ({ survey }) => {
+            posthog.capture('survey launched', {
+                name: survey.name,
+                id: survey.id,
+            })
+        },
```

Comment:
this is already captured backend
https://github.com/PostHog/posthog/blob/1228113aa06e3cf1a02f71613202e5a6228902f9/posthog/api/survey.py#L579
and we should avoid frontend captures because they are ad blocked if ad blockers are enabled, unless its not possible of course

Reply:
ohhh makes sense, i'll remove